src/code/z_player_lib.c

```diff
@@ -524,9 +586,55 @@ s32 func_8012364C(PlayState* play, Player* player, s32 arg2) {
                                                           : ITEM_NONE;
 }
 
-#pragma GLOBAL_ASM("asm/non_matchings/code/z_player_lib/func_80123810.s")
+u16 sCItemButtons[] = { BTN_CLEFT, BTN_CDOWN, BTN_CRIGHT };
```

Comment:
If this array included `BTN_CUP` then it would be fine, but it is used for checking the C buttons which can hold an item, that's why I wanted to use `Item` explicitly in the name.
I guess we could rename it and just add a comment about it missing C up

Reply:
Fair point, I'll think about this more

Reply:
How about `sActionButtons`?

Reply:
A is usually called the action button (it has the "action icon" as Navi calls it), which is associated to DoActions. It should probably also explicitly exclude B, since a lot of code treats B as like another C button. I think `sCItemButtons` is probably best out of the options discussed so far.